README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 | Language      | Server Application | Client Application (OpenAPI) |
 | ------------- | ------------------ | ---------------------------- |
-| Python        | Yes                | No                           |
-| Java / Groovy | Yes                | No                           |
+| Python        | Yes                | Yes                           |
+| Java / Groovy | Yes                | Yes                           |
 | JavaScript    | No                 | Yes                          |
 | gRPC          | -                  | Yes                          |
 
-## Running Deephaven
+## Run Deephaven
+
+This section is a quick start guide for running Deephaven from pre-built images.
+For detailed instructions on running Deephaven, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
+
+For detailed instructions on building and running Deephaven from source code, 
+see [Build and launch Deephaven](https://deephaven.io/core/docs/tutorials/launch-build).
+
+If you are not sure which of the two is right for you, you want to use the pre-built images.
+The pre-built images are for normal Deephaven deployments.
+Building from source is for developers that are modifying the Deephaven Community Core source code.
 
 ### Required Dependencies
 
-Building and running Deephaven requires a few software packages.
-1. `git`
-2. `java`
-3. `docker`
-4. `docker-compose`
-5. (Windows) [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+Running Deephaven requires a few software packages.
+
+| Package | Version | OS  |
+| ------- | ------- | --- |
+| docker  | ≥19.3 | All |
+| docker-compose | ≥1.29 | All |
```

Comment:
I wouldn't be against using semver ranges if we need to be more specific in the versions. Probably not needed at this point though unless we know we can get value from them

https://www.telerik.com/blogs/the-mystical-magical-semver-ranges-used-by-npm-bower

Reply:
The problem is that we don't know the ranges. Right now, we only know that if you use versions less than this things don't work.

Reply:
Gotcha, works for me